conda-recipes/pygdf/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - setuptools
     - libgdf 0.1.0a3.*
     - libgdf_cffi 0.1.0a3.*
-    - pandas 0.20.*
+    - pandas>=0.23.4
```

Comment:
I think we should make this >=0.20.3 since we know we work with that version and there may be other incompatibilities where a user doesn't want the latest and greatest pandas.